src/commons/error.rs

```diff
@@ -281,10 +321,11 @@ impl fmt::Display for Error {
             Error::PostTooBig => write!(f, "POST body exceeds configured limit"),
             Error::PostCannotRead => write!(f, "POST body cannot be read"),
             Error::ApiInvalidCredentials(e) => write!(f, "Invalid credentials: {}", e),
-            Error::ApiAuthError(e) => write!(f, "Authentication error: {}", e),
+            Error::ApiLoginError(e) => write!(f, "Login error: {}", e),
```

Comment:
See above comment about "login error" being too generic and overlapping with other errors, e.g. "invalid credentials". Or is this error supposed to be generic and covering cases not covered by more specific errors?

Reply:
See the above comment